samples/characterization/randomized-benchmarking/randomized-benchmarking.ipynb

```diff
@@ -126,7 +123,18 @@
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
```

Comment:
Same question here, about if the error is intended

Reply:
Same here as well; these could be resolved by fixes to QInfer.